### ISSUE TITLE
Possibility for donations?

### ISSUE THREAD

**Issue by underyx** [stevearc/pypicloud#159]
Hey, @stevearc, you've been quite responsive as a maintainer and we at @kiwicom really appreciate your work. I was thinking I'd ask our finance people if we can support pypicloud with something like $50/month. Would you care to set up a way for us to donate for that? Should I go for it?

**Comment by stevearc** [comment]
Of course, right after you post this issue I disappeared for a month 😛

This is a really thoughtful gesture, given that I haven't spent any time myself thinking about donations. Has your company done this before with other open source projects? If so, what platform(s) are you using for it?

**Comment by underyx** [comment]
Don't worry about the delay 🙃

Actually, no, we haven't.provided grants just out of the blue before. The only thing we've tried so far is www.kiwi.com/sourcelift, where we provide only flight tickets, not money. Do you have any suggestions for a platform to use?

**Comment by stevearc** [comment]
Oh, that's awesome! After a brief google it looks like two of the big players are [Patreon](https://www.patreon.com/), which is very generic, and [BountySource](https://salt.bountysource.com/) which is specifically for open source. Let me know if your finance guys go for it and I'll look into setting up a presence on one of those.

**Comment by mamoit** [comment]
Maybe [github sponsors](https://github.com/sponsors) can do the trick?

**Comment by stevearc** [comment]
Probably, but at this point I don't think it really fits how I think about pypicloud. For several years now I've basically just been maintaining it because some people find it useful and I like that something I made can provide utility for others, but I haven't really done any active development or pushed it forwards. Philosophically, I believe that sponsored projects should be under active development. I don't plan on changing how I maintain pypicloud anytime soon, so I'm going to keep it as-is with no donations.
